### ISSUE TITLE
Any keywords to get array item root properties in the formatting?

### ISSUE THREAD

**Issue by SergeyKokichev** [marchah/json-mapper-json#10]
Hello, I'm using this lib and I like it. But today I faced an interesting situation.

I have a json from Elasticsearch:
```javascript
{  
  "hits": {
    "total": 1,
    "hits": [
      {
        "_index": "some_index",
        "_type": "some_type",
        "_id": "123456",
        "_score": 1,
        "_source": {
          "id": 123456
        }
      }
    ]
  }
}
```
And I need to map it into json like this:
```javascript
{
   "type": "some_index/some_type"
   "id": "123456"
}
```

My mapping template:
```javascript
{	
	"hits": {
		path: "hits.hits",
		nested: {
			"id": {
				path: "_source.id"
			},
			"type": {
				path: "_type",
				formatting: (value) => { return value; }
			}
		}
	}
}
```

So the problem is that I can't get both values from _index and _type at the same time because they are root properties of an array item.

Not the best solution is to wrap each array item like this. And I won't use it
```javascript
"hits":[
   {
       "item": {
          "_index": "some_index",
          "_type": "some_type",
          "_id": "123456",
          "_score": 1,
          "_source": {
          "id": 123456
        }
    }
]
```

Is it possible to implement a new keyword like "$item" and it will be a whole array item, so it will be possible to grab any number of root properties at the same time.

Hope it's clear :)
Thanks in advance.

**Comment by marchah** [comment]
Hi,

It should work, surprisingly very easy (too easy ?) try with this branch: https://github.com/marchah/json-mapper-json/tree/item-key-word

Your template
```javascript
{
  "hits": {
    path: "hits.hits",
    nested: {
      "id": {
        path: "_source.id"
      },
      "type": {
        path: "$item",
        formatting: (value) => (`${value._index}/${value._type}`),
      }
    }
  }
}
```

unit test: https://github.com/marchah/json-mapper-json/blob/item-key-word/test/json-mapper.spec.js#L1188

Let me know if it's working and I will merge it

**Comment by SergeyKokichev** [comment]
Hi,
I have tried the branch with $item and it works great!
Thank you!

**Comment by SergeyKokichev** [comment]
Hi,

$item works great.

Thanks!

**Comment by marchah** [comment]
Good, published in version `1.1.0`
